### DISCUSSION TITLE
More App Ideas

### DISCUSSION THREAD

**Discussion by jetzalel** [engageub/InternetIncome#167] [Ideas]
I wanted to ask if it was possible to add these:
[Dawn](https://www.dawninternet.com/)
[Wipter](wipter.com/register?via=BFF9B84F03)

**Comment by engageub** [comment]
Hello,

Thank you for sharing these new apps.
If there are docker containers for the apps, they can be added to the script.
Dawn app uses a chrome extension, so, you can use [test branch](https://github.com/engageub/InternetIncome/tree/test) with custom chrome browser.
Wipter App does not support linux at the moment.

Could you please confirm if these apps are actually paying?

Thank you

**Reply by mrjkjxyz** [reply]
Dawn test branch usage, how to handle account and password verification?

**Reply by engageub** [reply]
Hello,
If you set `CUSTOM_CHROME=true` in `properties.conf` file in [test branch](https://github.com/engageub/InternetIncome/tree/test), you can use browser similar to your desktop chrome browser.
You have to access the browser using the URLs mentioned and then install the extensions in the browser.

Accessing Browser URLS: https://github.com/engageub/InternetIncome/wiki/Frequently-Asked-Questions#-how-to-access-127001-urls-if-gui-is-not-available-in-linux-

Thank you

**Comment by engageub** [comment]
Hello,
Wipter has been added to all the branches now.
Please download the latest version of InternetIncome Script.

Thank you

**Comment by TradeBW** [comment]
Is Wipter any better now ? I heard complaints of not getting enough traffic on them, Is the issue still there ?

**Reply by engageub** [reply]
Hi,
Wipter restricts account access when logins are made from data centers or when the proxy service it employs identifies an IP address as non-residential. Regarding the traffic, it depends on demand.  You will also have to check the recent communication in their discord group. 

Thank you
